client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -186,6 +308,67 @@ export default function CampaignItemDetails( props: Props ) {
 	const deliveryEstimateFormatted = getCampaignEstimatedImpressions( display_delivery_estimate );
 	const campaignTitleFormatted = title || __( 'Untitled' );
 	const campaignCreatedFormatted = moment.utc( created_at ).format( 'MMMM DD, YYYY' );
+	const objectiveFormatted = ( () => {
+		switch ( objective ) {
+			case 'traffic':
+				return (
+					<>
+						<span> { trafficIcon() } </span>
+						<span>
+							{ translate( 'Traffic' ) }
```

Comment:
It would look better if we use bold fonts for the objective. It will make it clear what the purpose is and what the description for that objective is. But, this is just my opinion.

Reply:
i tried the bold, but it is not very nice (because it looks more prominent even from the headers)

I added a rule to make it darker gray and it looks better I think